ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.18__periodics.yaml

```diff
@@ -207,6 +207,34 @@ tests:
     - env: OPENSTACK_CLOUD
       resource_type: openstack-vexxhost-quota-slice
     workflow: hypershift-openstack-conformance
+- as: e2e-openstack-csi-cinder
+  minimum_interval: 72h
+  steps:
+    cluster_profile: hypershift
+    env:
+      CLUSTER_TYPE_OVERRIDE: openstack-vh-mecha-central
```

Comment:
do they need to run on mecha-central? Other environments might be less loaded.

Reply:
yes, CSI tests don't run fine on VEXXHOST yet. We'll take care of that.